cli/cli/commands/clean/clean.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/kurtosis-tech/kurtosis/cli/cli/command_str_consts"
 	"github.com/kurtosis-tech/kurtosis/cli/cli/out"
 	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface"
-	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/container_status"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/container"
```

Comment:
Encapsulate the current container status and new attributes: image name, args... in a new container object.

Reply:
@laurentluce I don't assume this is a breaking change?

Reply:
It is not a breaking change.